zendesk/resource_zendesk_organization.go

```diff
@@ -112,7 +112,7 @@ func unmarshalOrganization(d identifiableGetterSetter) (client.Organization, err
 	}
 
 	if v, ok := d.GetOk("group_id"); ok {
-		org.GroupID = v.(int64)
+		org.GroupID = int64(v.(int))
```

Comment:
Does all of this organization / target related code belong in this pr? should we split those changes off into their own?

Reply:
You're right, they don't really belong here - I just lazily lumped them into this PR. They aren't essential, just a bit of cleanup. I'll remove them from here.